pywb/static/wombat.js

```diff
@@ -2856,7 +2856,72 @@ var _WBWombat = function($wbwindow, wbinfo) {
         wb_funToString.apply = orig_apply;
     }
 
-
+    function initTimeoutIntervalOverrides($wbwindow) {
+        // because [setTimeout|setInterval]('document.location.href = "xyz.com"', time) is legal and used
+        // declare the beginning wrapper string first so we do not redeclare it multiple time in our override (closure)
+        var wrapper = 'var _____WB$wombat$assign$function_____ = function(name) {return (self._wb_wombat && self._wb_wombat.local_init &&self._wb_wombat.local_init(name)) || self[name]; };\n' +
```

Comment:
To avoid duplication, could this be combined with what's happening in `rewrite_script()`?
Maybe we can factor out the wrapper creation into a separate function, `wrap_script_js_proxy()` that computes prefix, like in rewrite_script on first use,
and then applies it to a string? Then, this function and rewrite_script() could use wrap_script_js_proxy()?

A little more refactoring, but I think will be worth it in keep this more maintainable :)